packages/crayons-core/src/components/form/readme.md

```diff
@@ -291,6 +291,72 @@ fields: [ // Each item in this array corresponds to a crayons input component.
 }
 ```
 
+## Text and Value indicators for select component choices
+
+The choices for select component(dropdown/multi_select) can follow any of the below formats.
+
+1. Using field_options object. Pass the key name that needs to be displayed as the dropdown option's text using, 'option_label_path' and the key name that needs to to be processed in the backend using, 'option_value_path'.
+
+```
+{
```

Comment:
can you add a working example for using `field_options` with `option label and value path` ?

Reply:
added